views/js/pciCreator/dev/audioRecordingInteraction/runtime/audioRecordingInteraction.js

```diff
@@ -72,6 +72,14 @@ define([
         _filePrefix: 'audioRecording',
         _recording: null,
         _recordsAttempts: 0,
+        _delayCallback: null,
+
+        _cleanDelayCallback: function _cleanDelayCallback() {
+            if (this._delayCallback) {
+                clearTimeout(this._delayCallback);
+                this._delayCallback = null;
+            }
+        },
```

Comment:
Isn't possible to use a closure instead?

Reply:
why, exactly?

Reply:
To avoid to pollute the object by exposing private data. Anyway, this is not a big deal here. If it is too complex to change that, no worries, this is good enough.